src/main/webapp/app/shared/metis/posting-content/posting-content-part/posting-content-part.components.ts

```diff
@@ -54,10 +54,16 @@ export class PostingContentPartComponent implements OnInit {
         private accountService: AccountService,
     ) {}
 
-    ngOnInit() {
+    ngOnInit(): void {
         this.processContent();
     }
 
+    ngOnChanges(changes: SimpleChanges): void {
+        if (changes['postingContentPart'] && !changes['postingContentPart'].firstChange) {
```

Comment:
```suggestion
        if (changes.postingContentPart && !changes.postingContentPart.firstChange) {
```
We mostly use this style in client code